distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -221,6 +222,27 @@ Complete the following procedure now that several queries have been executed.
 Based on the browser's configuration, the .xls file will be downloaded or automatically displayed in Excel.
 ====
 
+=== Configuration Export/Import
+==== Export Configuration Settings
+
+1) Launch DDF.
+
+2) At the Admin Console, select any Active Application that you want to save.
+
+3) For each Active Application selected, click Save changes on the configuration that you want to save. This creates the configuration files.
```

Comment:
If we end up keeping this, we shouldn't say that `This creates the configuration files`. Saving the configurations only applies the changes. The files don't get created until the `config-export` command is run.